### PR TITLE
debian/rules : Introduced options for configure

### PR DESCRIPTION
**Pull request by garyachy**

### CODE REVIEW

**Comment by stcheng** [comment]
Will the arguments passed to ./configure become CONFIGURE_ARGS?

**Comment by stcheng** [comment]
@garyachy

File: debian/rules [hunk 1]
```diff
@@ -28,7 +28,7 @@ include /usr/share/dpkg/default.mk
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
-	dh_auto_configure --
+	dh_auto_configure -- ${CONFIGURE_ARGS}
```
Comment (qiluo-msft) [inline debian/rules]: Where is it from? What problem are you solving? Give a sample usage?

Comment (garyachy) [inline debian/rules]: The idea was to be able generate sonic-swss binaries with debug symbols enabled.

The usage will be as follows.
export CONFIGURE_ARGS=--enable-debug=yes
cd sonic-buildimage/src/sonic-swss
make

Comment (qiluo-msft) [inline debian/rules]: You can do the same thing with commands:

    cd sonic-buildimage/src/sonic-swss
    ./configure --enable-debug=yes
    make

Do you feel this one meet your requirement?

**Comment by lguohan** [comment]
Enabled debug symbol #151, I feel we can close this one.

**Comment by garyachy** [comment]
Sure, just describe how to use this debug package.
Is it loaded to container automatically?

**Pull request closed without merging**